packages/core/src/global/responses/RestResponse.ts

```diff
@@ -66,11 +66,11 @@ export class RestResponse<Body = unknown> {
   }
 
   get body(): Body {
-    if (this._body === null) {
-      throw new AppError('Response has no body')
+    if (this._errorMessage) {
+      throw new AppError('Response is a failure')
```

Comment:
"Response is a failure" não faz sentido em inglês. Você pode usar "Response failed".